xpcPlugin/MessageHandlers.cpp

```diff
@@ -626,6 +630,89 @@ namespace XPC
 			}
 		}
 	}
+	
+	void MessageHandlers::HandleGetT(const Message& msg)
+	{
+		const unsigned char* buffer = msg.GetBuffer();
+		std::size_t size = msg.GetSize();
+		if (size != 30)
+		{
+			Log::FormatLine(LOG_ERROR, "GTER", "Unexpected message length: %u", size);
+			return;
+		}
+		unsigned char aircraft = buffer[5];
+		Log::FormatLine(LOG_TRACE, "GTER", "Getting terrain information for aircraft %u", aircraft);
+		
+		double loc[3], X, Y, Z;
```

Comment:
This is really personal preference, but I don't care for this style of declaration. Please declare one thing per line.